### DISCUSSION TITLE
Adding Custom Supported Rules

### DISCUSSION THREAD

**Discussion by dev-federico** [dedoc/scramble#279] [Q&A] [answered]
Hello,

is it possible to add custom supported rules?
For example one of my body parameters is using "sometimes" and this is not supported, can I add it somehow?

**Comment by romalytvynenko** [comment, marked as answer]
@spaceproj support for custom rules (or for Laravel's not implmented rules) is coming in next bigger version. You can track progress here: https://github.com/dedoc/scramble/pull/237

**Reply by dev-federico** [reply]
Thank you for your feedback and your great job!
